src/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ list(APPEND predict_public_header
   ${LIBPREDICT_INCLUDE_DIR}/predict/orbit.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/observer.h
   ${LIBPREDICT_INCLUDE_DIR}/predict/tle.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/juliandate.h
+  ${LIBPREDICT_INCLUDE_DIR}/predict/predict.h
```

Comment:
I agree. I mashed all public headers into one in commit ab1a0b92df554758c498adedc5f9d33491f4e448.